lib/http_server.ex

```diff
@@ -21,12 +23,12 @@ defmodule HTTPServer do
     request =
       socket
       |> read_line()
-      |> parse_request()
+      |> Request.parse_request()
 
     response =
       request
-      |> build_response()
-      |> format_response()
+      |> Router.router(&(Routes.route(&1)))
```

Comment:
this is Elixry to the point of confusion.  I see that production code will run the fixture Routes.  Is it possible to only use those routes when starting a server to integration tests? and use a default Routes for when you start the server with production code?

Reply:
Ooooh great idea. The only thing that could be challenging is that the server has to be started before the integration tests are called, so I may need to make a specific start task that spins up the server + routes for the integration test suite?

Reply:
I like that idea, can't wait to see if it works

Reply:
Let me know if the new `determine_routes` functionality is what you imagined here. I feel like there is probably a better way, but after a lot of trial and error I found this to be the easiest and quickest fix!

Reply:
Its super creative, I left a comment, but I like it!